src/Containers/Microsoft.NET.Build.Containers/Registry/HttpExtensions.cs

```diff
@@ -39,7 +39,7 @@ public static Uri GetNextLocation(this HttpResponseMessage response)
     internal static bool IsAmazonECRRegistry(this Uri uri)
     {
         // If this the registry is to public ECR the name will contain "public.ecr.aws".
-        if (uri.Authority.Contains("public.ecr.aws"))
+        if (uri.Authority.Contains(RegistryConstants.PublicAmazonElasticContainerRegistryDomain))
```

Comment:
I like this and a lot of similar improvements like it, thank you for that code cleanup. It looks great!

Reply:
Thanks! Lots of it was suggested by other @MichalPavlik, who is all about making me not make as many messes :D